platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheGracefulDegradationIntegrationTest.groovy

```diff
@@ -65,7 +146,8 @@ class ConfigurationCacheGracefulDegradationIntegrationTest extends AbstractConfi
         problems.assertResultConsoleSummaryHasNoProblems(result)
         problems.assertResultHtmlReportHasProblems(result) {
             totalProblemsCount = 1
-            withProblem("Build file 'build.gradle': line 5: invocation of 'Task.project' at execution time is unsupported.")
+            checkReportProblems = true
```

Comment:
❓ I think it should be set by default when we call `assertResultHtmlReportHasProblems` method. By using this method we're literally saying "console and html reports are going to be different", so I think it's fair to say that we do want to check html report problems each time.